src/assets/styles/main.scss

```diff
@@ -0,0 +1,21 @@
+@font-face {
+	font-family: 'Source Sans Pro';
+	font-style: normal;
+	font-weight: 400, 600, 700;
+	font-stretch: 100%;
+	font-display: swap;
+	src: url('/utils/fonts/SourceSansPro-Regular.ttf'), url('/utils/fonts/SourceSansPro-SemiBold.ttf'),
+	url('/utils/fonts/SourceSansPro-Bold.ttf')
+}
+*{
+	font-family: 'Source Sans Pro';
+	font-weight: 400;
+	font-style: normal;
+	padding: 0;
+	margin: 0;	
+}
+
+body{
+	padding: 0 !important;
```

Comment:
do u need important?

Reply:
Yes, because without that, padding still using